Masquerade4997Library/src/main/java/Library4997/MasqMotors/MasqMotor_Tests.java

```diff
@@ -8,87 +8,87 @@
-            throw new AssertionError("getAbsolutePositionTest Failed");
-    }
-     private void getVelocityTest() {
-        if (!(masqMotor.getVelocity(10, 1e9, 5) == 120)) throw new AssertionError("getVelocityTest Failed");
-    }
-     private void getAngleTest() {
-        if (masqMotor.getAngle(30, 12) != 1) throw new AssertionError("getAngleTest Failed");
-    }
-     private void setPowerTest() {
-        if (masqMotor.setPower(0.5) != 0.5) throw new AssertionError("setPowerTest Failed");
-    }
-     private void setVelocityTest() {
-         masqMotor.setClosedLoop(true);
-        if (Math.abs(masqMotor.setVelocity(0.5, 5, 100) - 1) > 0.1) throw new AssertionError("setVelocityTest Failed");
-         masqMotor.setClosedLoop(false);
-    }
-     private void getStalledTest() {
-        masqMotor.setStalledRPMThreshold(1);
-        if (!masqMotor.getStalled(10, 1e9, 1200)) throw new AssertionError("getStalledTest failed");
-    }
-    private void enableStallDetectionTest(){
-        masqMotor.setStalledRPMThreshold(1);
-        masqMotor.setUnStalledAction(() -> {throw new AssertionError("enableStallDetectionTest failed"); });
-        masqMotor.enableStallDetection(10,1e9,1200);
-    }
-    @Override
-    public void RunAll() {
-        RunWithoutEncoderTest();
-        RunUsingEncoderTest();
-        setDistanceTest();
-        setBreakModeTest();
-        unBreakModeTest();
-        getCurrentPositionTest();
-        getAbsolutePositionTest();
-        getVelocityTest();
-        getAngleTest();
-        setPowerTest();
-        setVelocityTest();
-        getStalledTest();
-        enableStallDetectionTest();
-    }
+  }
+  private void getAbsolutePositionTest() {
+    if (masqMotor.getAbsolutePosition() != masqMotor.motor.getCurrentPosition())
+      throw new AssertionError("getAbsolutePositionTest Failed");
+  }
+  private void getVelocityTest() {
+    if (!(masqMotor.getVelocity(10, 1e9, 5) == 120)) throw new AssertionError("getVelocityTest Failed");
+  }
+  private void getAngleTest() {
+    if (masqMotor.getAngle(30, 12) != 1) throw new AssertionError("getAngleTest Failed");
+  }
+  private void setPowerTest() {
+    if (masqMotor.setPower(0.5) != 0.5) throw new AssertionError("setPowerTest Failed");
+  }
+  private void setVelocityTest() {
+    masqMotor.setClosedLoop(true);
+    if (Math.abs(masqMotor.setVelocity(0.5, 5, 100) - 1) > 0.1) throw new AssertionError("setVelocityTest Failed");
+    masqMotor.setClosedLoop(false);
+  }
+  private void getStalledTest() {
+    masqMotor.setStalledRPMThreshold(1);
+    if (!masqMotor.getStalled(10, 1e9, 1200)) throw new AssertionError("getStalledTest failed");
+  }
+  private void enableStallDetectionTest(){
+    masqMotor.setStalledRPMThreshold(1);
+    masqMotor.setUnStalledAction(() -> {throw new AssertionError("enableStallDetectionTest failed"); });
+    masqMotor.enableStallDetection(10,1e9,1200);
+  }
+  @Override
+  public void RunAll() {
```

Comment:
We should find a better way. Maybe an annotation? So like @Test and then it loops through all instances of @Test? Not sure. This, currently, doesn't seem like a solution that is scalable.